components/dom_crawler.rst

```diff
@@ -333,10 +349,18 @@ and :phpclass:`DOMNode` objects::
         // avoid the exception passing an argument that html() returns when node does not exist
         $html = $crawler->html('Default <strong>HTML</strong> content');
 
+        $html = $crawler->outerHtml();
```

Comment:
lets move this below the versionadded of `html()` method, moving it away from the context makes not so much sense and Introduce the outherHtml()` method with a short sentence after the `html()` versionadded.

To be clear regarding the order:

* `html()` code example
* `html()` versionadded directive
* `outherHtml()` introduction/description
* `outherHtml()` code example
* `outherHtml()` versionadded directive

Thanks.

Reply:
Ok got it :+1: thanks.

Is it better now ?